pkg/email/system_intake_change_lcid_retirement_date_test.go

```diff
@@ -122,7 +127,7 @@ If you have questions, please contact the Governance Team at <a href="mailto:%s"
 		*decisionNextSteps.StringPointer(),
 	)
 
-	s.Run("Should omit additional info if absent", func() {
+	s.Run("Should omit additional info and issuedAt if absent", func() {
```

Comment:
Combo test instead of separate test? I'll let it slide... this time!

Reply:
Yeah, I considered the same, but since there's no branching logic between the two cases and they don't affect each other I thought it was reasonable enough 😁